functions/snippets/src/main/java/com/example/functions/OcrTranslateApiMessage.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.functions;
+
+// [START functions_ocr_translate_pojo]
+import com.google.gson.Gson;
+import com.google.protobuf.ByteString;
+import com.google.pubsub.v1.PubsubMessage;
+import java.util.Base64;
+
+// Object for storing OCR translation requests
+public class OcrTranslateApiMessage {
+  private String text;
+  private String filename;
+  private String lang;
+
+  private static final Gson gson = new Gson();
+
+  public OcrTranslateApiMessage(String text, String filename, String lang) {
+    this.text = text;
+    this.filename = filename;
+    this.lang = lang;
+  }
+
+  public String getText() {
+    return text;
+  }
+
+  public String getFilename() {
+    return filename;
+  }
+
+  public String getLang() {
+    return lang;
+  }
```

Comment:
Why?

Reply:
@kurtisvg has a preference for getters/setters - but I can drop these if you'd prefer.

Reply:
Your not using them outside your class, so I'd prefer dropping them, but feel free to ask kurtis about my comment.

Reply:
Will do. Also, these _are_ used outside the class (e.g. [here](https://github.com/GoogleCloudPlatform/java-docs-samples/pull/2552/files#diff-7400aeba097e7758edf3c3f4cc8a9b7aR41-R43))

Reply:
Spoke with Les - we should use getters/setter from outside of the class (like where you linked) but make sure to use the internal reference (like in `fromPubSubData`).